include/utils.php

```diff
@@ -4501,6 +4481,9 @@ function str_split_php4($string, $length = 1)
     return $return;
 }
 
+/*
+ * WONTFIX: Not updating this.. This should be removed !!!
+ */
 if (version_compare(phpversion(), '5.0.0', '<')) {
```

Comment:
Hello @daniel-samson.
Maybe we could really delete this code, as its only purpose is to define the str_split function under PHP4. If you agree i'll take care of that.